src/wagtailtrans/models.py

```diff
@@ -101,6 +102,10 @@ class TranslatablePage(Page):
 
     is_creatable = False
 
+    search_fields = Page.search_fields + [
+        FilterField('language_id'),
```

Comment:
Let's merge this now to at least facilitate searching on the language. 
I've opened an issue #49 in which we can specify what we want to do with the search.